scripts/leapfrog/README.md

```diff
@@ -0,0 +1,136 @@
+# Leapfrog Documentation
+
+#### Overview
+A Leapfrog upgrade is a major upgrade that skips at least one release. Currently
+RPCO supports leapfrog upgrades from kilo to newton-14.1.
+
+#### Terms
+
+* [RPCO](https://github.com/rcbops/rpc-openstack): Rackspace Private Cloud powered by OpenStack
+* [OSA](https://github.com/openstack/openstack-ansible):  OpenStack Ansible
+* [OSA-OPS](https://github.com/openstack/openstack-ansible-ops):  OpenStack Operations
+* [Kilo](https://github.com/rcbops/rpc-openstack/tree/kilo): The RPCO release of OpenStack Kilo
+* [Newton-14.1](https://github.com/rcbops/rpc-openstack/tree/newton-14.1) The second RPCO release of OpenStack Newton. This version includes Leapfrog upgrade tools.
+
+## Executing a leapfrog upgrade
+
+The start point is a kilo deployment. It is useful to check the kilo deployment
+is healthy before starting the leapfrog process, for example by running tempest.
+
+```
+root@kilo:/opt/rpc-openstack/rpcd/playbooks# openstack-ansible ../../scripts/run_tempest.yml -t tempest_execute_tests -vv
```

Comment:
Ok with that too, but we should insist on the fact that a health check should be done.